### PR TITLE
Add test fixtures for Arrays.srv

### PR DESCRIPTION
**Pull request by jacobperron**
Connects to https://github.com/ros2/rosidl/pull/387

### CODE REVIEW

File: test_msgs/include/test_msgs/service_fixtures.hpp [hunk 1]
```diff
@@ -136,4 +139,79 @@ get_services_basic_types()
...
+get_services_arrays()
+{
+  std::vector<
+    std::pair<
+      test_msgs::srv::Arrays::Request::SharedPtr,
+      test_msgs::srv::Arrays::Response::SharedPtr
+    >
+  > services;
+  {
+    auto basic_types_msgs = get_messages_basic_types();
```
Comment (dirk-thomas) [inline test_msgs/include/test_msgs/service_fixtures.hpp]: Nitpick: maybe move it closer to where it is used the first time. When reading this I was puzzled at first what it is good for...

Same in Python below.

File: test_msgs/src/test_msgs/service_fixtures.py [hunk 2]
```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# TODO(jacobperron): Rename functions to match C++ equlivalents
```
Comment (dirk-thomas) [inline test_msgs/src/test_msgs/service_fixtures.py]: Spelling: equlivalents

File: test_msgs/src/test_msgs/service_fixtures.py [hunk 3]
```diff
@@ -97,11 +100,58 @@ def get_msg_basic_types():
...
+    return srvs
+
+
 def get_test_srv(service_name):
     if 'Empty' == service_name:
         srv = get_msg_empty()
     elif 'BasicTypes' == service_name:
         srv = get_msg_basic_types()
+    elif 'Arrays' == service_name:
+        srv = get_msg_arrays()
```
Comment (dirk-thomas) [inline test_msgs/src/test_msgs/service_fixtures.py]: Nitpick: alphabetical order might make more sense here (would also swap the two existing ones).

**Review by dirk-thomas — APPROVED** [review]

File: test_msgs/src/test_msgs/service_fixtures.py (hunk 3 above)
Comment (jacobperron) [reply test_msgs/src/test_msgs/service_fixtures.py]: 2f0076c

File: test_msgs/src/test_msgs/service_fixtures.py (hunk 2 above)
Comment (jacobperron) [reply test_msgs/src/test_msgs/service_fixtures.py]: 2f0076c

File: test_msgs/include/test_msgs/service_fixtures.hpp (hunk 1 above)
Comment (jacobperron) [reply test_msgs/include/test_msgs/service_fixtures.hpp]: 2f0076c + 654a7ea

**Comment by jacobperron** [comment]
Re-running CI just in case.
Linux: [Build Status](https://ci.ros2.org/job/ci_linux/7628/)

**Pull request merged**